conn-sdk-cli/README.md

```diff
@@ -0,0 +1,98 @@
+# conn-sdk-cli
+
+`conn-sdk-cli` is a CLI tool that helps with development of connectors. To
+install the tool, run:
+
+```shell
+go install github.com/conduitio/conduit-connector-sdk/conn-sdk-cli@latest
+```
+
+For more information about using the available commands, run:
+
+```shell
+conn-sdk-cli help
+```
+
+or the following for a specific command:
+
+```shell
+conn-sdk-cli <command> --help
+```
+
+The following sections describe the most important commands in more details.
+
+## readmegen
+
+The `readmegen` command is used to generate sections in a README file that
+document the connector. `readmegen` searches for certain tags in the README file
+and replaces them with appropriate values. For example, if
+`<!-- readmegen:name -->` is found in a README, then `readmegen` will replace it
```

Comment:
I tested the command out, and I think mentioning the flag `--write` will be helpful here, the user will mostly need it, took me a while to find the option to overwrite

Reply:
will merge so we have a link for the blog post, this can be added later

Reply:
@hariso